arangod/Indexes/IndexIterator.h

```diff
@@ -285,6 +285,9 @@ struct IndexIteratorOptions {
   bool evaluateFCalls = true;
   /// @brief enable caching
   bool enableCache = true;
+  /// @brief number of lookahead elements considered before computing the next
+  /// intersection of the Z-curve with the search range
+  size_t lookahead = 2;
```

Comment:
And here we have a default value of 2. This can be confusing.

Reply:
made 1 as well

Reply:
Now the default is everywhere 1.